Authy2FA/Controllers/ManageController.cs

```diff
@@ -127,7 +127,7 @@ public ActionResult AddPhoneNumber()
                 user.AuthyId = result.UserId;

```

Comment:
I couldn't agree more, if `user.AuthyId` is `null` we're essentially creating an invalid user, and definitely that is not good. Please have a look at f448153.